backend/package.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "learning-drive",
+    "version": "1.0.0",
+    "description": "Learning drive social media application",
+    "main": "dist/src/index.js",
+    "scripts": {
+        "build": "prisma generate && prisma migrate deploy && npx tsc",
+        "prestart": "npm run build",
```

Comment:
Are we using npm or yarn for the backend?

Reply:
Currently using npm since I haven't used yarn before, but don't really mind if we want to change this.